drivers/include/periph/uart.h

```diff
@@ -111,6 +111,44 @@ enum {
     UART_NOMODE     = -4    /**< given mode is not applicable */
 };
 
+/**
+ * @brief   Definition of possible parity modes
+ */
+#ifndef HAVE_UART_PARITY_T
+typedef enum {
+   UART_PARITY_NONE,   /**< no parity */
+   UART_PARITY_EVEN,   /**< even parity */
+   UART_PARITY_ODD,    /**< odd parity */
+   UART_PARITY_MARK,   /**< mark parity */
+   UART_PARITY_SPACE   /**< space parity */
+} uart_parity_t;
+#endif
+
+/**
+ * @brief   Definition of possible data bits lengths in a UART frame
+ */
+#ifndef HAVE_UART_DATA_BITS_T
+typedef enum {
+    UART_DATA_BITS_8,   /**< 8 data bits */
```

Comment:
why not put these in some coherent order? Don't care if incremental or decremental, but some consistent order :-)

Reply:
ACK